spec/lib/common/models/base_spec.rb

```diff
@@ -0,0 +1,7 @@
+# frozen_string_literal: true
+require 'rails_helper'
+require_dependency 'common/models/base'
+
+# FIXME: Add this spec
+describe Common::Base do
+end
```

Comment:
it will be in a separate PR. Currently have close to 100% coverage, but it's based off of Prescriptions. I think it should be more generic than that, but didn't want to spend umpteen hours rewriting specs when it's already at 100%